src/bundle-manifest.ts

```diff
@@ -250,7 +253,10 @@ export function generateShellMergeStrategy(
     generateEagerMergeStrategy(shell),
     generateMatchMergeStrategy(
         (b) => b.files.has(shell) ||
-            b.entrypoints.size >= minEntrypoints && !getBundleEntrypoint(b))
+            b.entrypoints.size >= minEntrypoints && !getBundleEntrypoint(b)),
```

Comment:
Lines 254-260 are realllllly hard to grok, can you simplify? Add curly brackets for the arrow function  `(b) => { }`? Name `b`? Define these two functions before passing them to `generateMatchMergeStrategy`?